tests/integration_tests/test_custom_components.py

```diff
@@ -33,6 +33,9 @@
 
 @dataclass
 class CustomTestCombinerConfig(BaseCombinerConfig):
+
+    type: str = "custom_combiner"
```

Comment:
Smaller/separate PR suggestion: This could be its own PR.

Reply:
Agreed, will factor this one out!

Reply:
Per our standup discussion, factored out into this PR: https://github.com/ludwig-ai/ludwig/pull/2627